include/unix/osd.h

```diff
@@ -73,6 +73,10 @@
 	(((err) == EAGAIN)	||		\
 	 ((err) == EWOULDBLOCK))
 
+#define OFI_SOCK_TRY_ACCEPT_AGAIN(err)		\
+	(((err) == EAGAIN)	||		\
+	 ((err) == EWOULDBLOCK))
```

Comment:
This needs to be defined in windows/osd.h, possibly the same as TRY_SND_RCV there.  That I'm not sure of.

Reply:
I have defined the macro in `windows/osd.h` as follows:
```
#define OFI_SOCK_TRY_ACCEPT_AGAIN(err)         \
       (((err) == EAGAIN)      ||              \
        ((err) == EWOULDBLOCK))
```

The macro `OFI_SOCK_TRY_SND_RCV_AGAIN` for Windows handles the error `ETIMEDOUT` but I don't see the error listed in https://docs.microsoft.com/en-us/windows/win32/api/winsock2/nf-winsock2-accept. As a result, I decided to remove it from `OFI_SOCK_TRY_ACCEPT_AGAIN`.

Disclaimer: I haven't tested the patch on Windows.